src/style.js

```diff
@@ -130,7 +131,7 @@ const style = css`
     content: '';
   }
   ha-card[artwork*='full-cover'].--has-artwork .mmp-player {
-    background: linear-gradient(to top, var(--mmp-overlay-color) 25%, transparent 100%);
+    background: linear-gradient(to top, var(--mmp-overlay-color) var(--mmp-overlay-color-stop), transparent 100%);
```

Comment:
Same as above

Reply:
I mixed both PR in the same branch. I removed the related commit.